Formula/libsolv.rb

```diff
@@ -0,0 +1,60 @@
+class Libsolv < Formula
+  desc "Library for solving packages and reading repositories"
+  homepage "https://github.com/openSUSE/libsolv"
+  url "https://github.com/openSUSE/libsolv/archive/refs/tags/0.7.23.tar.gz"
+  sha256 "0286155964373c6cc3802d025750786c3ee79608d5cb884598e110e3918bb2fe"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "xz"
+  depends_on "zstd"
+
+  uses_from_macos "bzip2"
+  uses_from_macos "expat"
+  uses_from_macos "zlib"
+
+  def install
+    args = %w[
```

Comment:
```suggestion
    args = %W[
```
so you don't need to add `CMAKE_INSTALL_RPATH` separately.